### ISSUE TITLE
Clicking on Pocket widget doesn't launch the pocket login page, Browser restart opens pocket login (intermittent issue) 

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#6586]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
Clicking on Pocket widget doesn't launch the pocket login page

**Expected behavior:**
Should open pocket login page in a new tab when clicked on the pocket widget 

- Platform (Win7, 8, 10? macOS? Linux distro?):
Windows 10 x64

- Brave Version (revision SHA):
Brave	0.13.0
rev	0c7a64d
Verified on 0.12.15 and works properly

- Steps to reproduce:
    1. Enable Pocket widget from Advance settings
    2. (Imp step) Do not login when the pocket login page comes up after enabling
    3. Close browser and restart, (_Pocket login page may come up again_) 
    4. Click on the pocket widget, doesn't launch login page in a new tab, console shows error
    5. Clear browsing history by pressing Ctrl+Shift+Del, select all switches and click save (any pref change restart should work but I have only seen this happen on clear browser data, but not on all instances)
    6. Click Yes in the pref restart prompt
    7. After browser restarts pocket login may come up(intermittent issue)
      
    Pocket login page showing up after restart is an intermittent issue can't consistently reproduce but clicking on the pocket widget always shows error on console. (both scenarios captured separately) 

- Screenshot if needed:
![pocket1](https://cloud.githubusercontent.com/assets/17010094/21769599/66068d5c-d677-11e6-9bdb-8483564bb27b.gif)
![pocket 2](https://cloud.githubusercontent.com/assets/17010094/21769601/6958bef8-d677-11e6-8bd4-6077d4b37e16.gif)

- Any related issues:
cc: @darkdh  @jonathansampson

**Comment by bbondy** [comment]
Especially since this is intermittent, it'd be super helpful to share what the console message is, thx!

**Comment by bbondy** [comment]
Oh I think I see it in the GIF, here it is for easier reading:
<img width="724" alt="screenshot 2017-01-15 11 44 29" src="https://cloud.githubusercontent.com/assets/831718/21964341/02e0065e-db18-11e6-8fe9-7c74c171a2c5.png">

**Comment by bbondy** [comment]
Which comes from here:
```
void AtomExtensionHostDelegate::CreateTab(content::WebContents* web_contents,
                                            const std::string& extension_id,
                                            WindowOpenDisposition disposition,
                                            const gfx::Rect& initial_rect,
                                            bool user_gesture) {
  NOTIMPLEMENTED();
}


```

**Comment by bbondy** [comment]
I think this is fixed with the latest preview. @srirambv please verify with Preview 8 and re-open if you can reproduce. Thanks!

**Comment by srirambv** [comment]
Working on Preview 8. Clicking the widget opens pocket login page

**Comment by srirambv** [comment]
>Especially since this is intermittent, it'd be super helpful to share what the console message is, thx!
@bbondy  The intermittent issue is that the login page used to come up randomly on restart. Will check if the issue is still happening on preview 8 and update the issue

**Comment by luixxiul** [comment]
@srirambv will you set QA steps more specifically? The steps to reproduce the error seem to me too complex to follow. If you can no longer reproduce it at all, please remove the milestone. thanks

**Comment by srirambv** [comment]
@luixxiul The main issue was clicking on pocket widget was not loading the login page which has been resolved in 0.13.0 preview 8 onward. The intermittent issue is with respect to login page showing up when you have not logged in after enabling pocket which I think is also fixed as the login page opens when the widget is clicked.
